resources/tests/msr/msr_list_T2A_AMD_MILAN_6.1host_5.10guest.csv

```diff
@@ -8447,7 +8447,7 @@ MSR_ADDR,STATUS,VALUE
 0xc00000fd,unimplemented,0x0
 0xc00000fe,unimplemented,0x0
 0xc00000ff,unimplemented,0x0
-0xc0000100,implemented,0x7f9c115aa500
```

Comment:
Since this would be 1.4MM lines... could we drop all the unimplemented lines and only keep implemented ones? That would greatly reduce the content of those files (40MB after this change now)

```sh
% grep -v unimplemented *.csv |wc -l
7716
pablob@uf0d78c75c44b51 data/msr
% grep unimplemented *.csv |wc -l
1394160
```

Reply:
> could we drop all the unimplemented lines and only keep implemented ones?

That's great idea. We must do it.

Reply:
done